src/libaktualizr/utilities/utils.cc

```diff
@@ -798,7 +799,7 @@ TemporaryFile::TemporaryFile(const std::string &hint)
 
 TemporaryFile::~TemporaryFile() { boost::filesystem::remove(tmp_name_); }
 
-void TemporaryFile::PutContents(const std::string &contents) {
+void TemporaryFile::PutContents(const std::string &contents) const {
```

Comment:
There's something unintuitive about a function that makes system calls (`write` no less!) being `const`, but I won't complain about more `const`s!